docs/install-docker.md

```diff
@@ -12,7 +12,7 @@ To make adjustments to the way the application runs, edit and review [docker-com
 
 The UI and API containers have been set to run on start up to install / update dependencies.
 
-Set up the [front-end ui client](ui/README.md) or [back-end api server](api/README.md) as needed.
+Set up the [front-end ui client](/ui/) or [back-end api server](/api/) as needed.
```

Comment:
Why was the `README.md` removed?

Reply:
I was simply suggesting that the .md files should still be referenced in the diff above. Should the diff have `/docs/ui/index.md` instead of `/ui/`?